docs/modules/rediscache.md

```diff
@@ -1,22 +1,21 @@
 # [Azure Redis Cache](https://azure.microsoft.com/en-us/services/cache/)
 
-_Note: This module is EXPERIMENTAL. To enable this module, you must run Open Service Broker for Azure with modules.minStability set to `experimental`_
+_Note: This module is EXPERIMENTAL and is not included in the General Availability release of Open Service Broker for Azure. It will be added in a future OSBA release._
```

Comment:
Why does this remove the instructions on how to enable the module?

Reply:
Actually I haven't changed this section... I think something went wrong with my version control tool or text editor...